docs/site_organization/working_with_page.md

```diff
@@ -117,6 +118,18 @@
         Settings and function of custom-made blocks of content depend on their design.
         For details on adjusting and using them, contact your website administrator.
 
+    ### Targeting block
+    
+    In the **Basic** tab, perform the following actions:
```

Comment:
Do you have to perform the actions below in a specific order? Is so, I would make this a numbered list. A bulleted list implies that actions can be performed in random order.

Reply:
No specific order necessary. The structure is the same as with other blocks, did not want to introduce a different organization.